### DISCUSSION TITLE
Tiptap vue2 - Mention / Suggestion error

### DISCUSSION THREAD

**Discussion by GitPAV** [ueberdosis/tiptap#2495] [General Questions] [answered]
Hello and thanks you for your amazing project !
Sorry if i'm posting in an inappropriate section.

I'm using tiptap with vue2 https://www.npmjs.com/package/@tiptap/vue-2
I'm trying to use mention with titap vue2 https://www.npmjs.com/package/@tiptap/extension-mention

All package are up to date but i can't make it work, i got that error when i try to run serve :

<img width="286" alt="error" src="https://user-images.githubusercontent.com/47319615/152777137-3b35759a-f6ef-47f7-8656-92a1cfd4f5cf.png">

(And big console error also : `TypeError: Object(...) is not a function`
tiptap-suggestion l7:  `const escapedChar = escapeForRegEx(char);`

Do you have any idea on how to fix this ? 
I'm struggling since a few day to make it work 

Here is my tiptap package list :

<img width="167" alt="ookok" src="https://user-images.githubusercontent.com/47319615/152777487-06cae141-0830-45a9-892f-348bb40341cd.png">

**Comment by philippkuehn** [comment, marked as answer]
You should update all these Tiptap packages.

**Reply by GitPAV** [reply]
Hello, thanks for your reply :)

Do you mean update to vue 3 ? It's not possible for the moment (i'm working with a team)

TiptapVue2 & tiptapMention are up to date (or i'm missing something)
<img width="565" alt="mentionVue" src="https://user-images.githubusercontent.com/47319615/152807085-51ef8a1c-c05e-4d04-abf8-c1adc1415174.png">
<img width="572" alt="vue2" src="https://user-images.githubusercontent.com/47319615/152807090-30810454-0ff1-40c3-bb74-c355e38272b5.png">

**Reply by philippkuehn** [reply]
No, just update all used tiptap packages to the latest version.

For example `@tiptap/starter-kit@2.0.0-beta.94` -> `@tiptap/starter-kit@2.0.0-beta.183` (wow 100 versions 😅)

**Reply by GitPAV** [reply]
Sorry, I wasn't aware of that 😢 

Just tried, it fixed escapeForRegex error but add new errors.
We have tiptap legacy code, we should update the whole thing, i don't think i can make suggestion work with our current packages version 

Thanks again for your time 😃
